lib/lfrfid/tools/t5577.c

```diff
@@ -2,20 +2,26 @@
 #include <furi.h>
 #include <furi_hal_rfid.h>
 
+// Required time to load configurations, must be >= 3ms
 #define T5577_TIMING_WAIT_TIME 400
-#define T5577_TIMING_START_GAP 30
-#define T5577_TIMING_WRITE_GAP 18
-#define T5577_TIMING_DATA_0 24
-#define T5577_TIMING_DATA_1 56
+
+// Required time to write block, typical value is around 5.6ms
 #define T5577_TIMING_PROGRAM 700
 
+// 1 / 125khz == 8 microseconds; Time = Cycles * 8
+#define CYCLES_TO_TIME(time) ((time) / 125000UL)
```

Comment:
At least that place is completely broken, anything less than 125000 will be equal 0.

Reply:
😱 thanks